Running python manage.py dumpdata grid.grid attached to terminal... up, run.21
python: can't open file 'manage.py': [Errno 2] No such file or directory
